plugins/standard_tests/newhelm/tests/simple_safety_tests.py

```diff
@@ -0,0 +1,140 @@
+from collections import defaultdict
+import csv
+from pydantic import BaseModel
+from typing import List, Dict, Mapping
+
+from newhelm.aggregations import mean_of_measurement
+from newhelm.annotators.llama_guard_annotator import (
+    LlamaGuardAnnotation,
+    LlamaGuardAnnotator,
+)
+from newhelm.base_annotator import BaseAnnotator
+from newhelm.base_test import BasePromptResponseTest, TestMetadata
+from newhelm.dependency_helper import DependencyHelper
+from newhelm.external_data import ExternalData, WebData
+from newhelm.placeholders import Prompt, Result, SUTOptions
+from newhelm.single_turn_prompt_response import (
+    MeasuredTestItem,
+    PromptWithContext,
+    TestItemAnnotations,
+    TestItem,
+)
+from newhelm.test_registry import TESTS
+from newhelm.typed_data import TypedData
+
+
+_HARM_AREAS = [
+    "Illegal and Highly Regulated items",
+    "Physical harm and Violence",
+    "Scams and fraud",
+    "Child safety",
+    "Suicide, Self-Harm, and Eating Disorders",
+]
+
+
+class SSTTestItemContext(BaseModel):
+    harm_area: str
+    prompt_category: str
+
+
+# TODO: Other languages?
+
+
+class SimpleSafetyTests(BasePromptResponseTest):
+    """
+    SimpleSafetyTests consists of 100 prompts over five severe harm-areas for the rapid identification of critical safety risk in an LLM.
+    The harm-areas are as follows:
+        - Illegal Items
```

Comment:
This is slightly different than the _HARM_AREAS above. Which is right? And once we've found out, can we remove the duplication?